app/src/test/java/com/wtmberlin/ui/EventsViewModelTest.kt

```diff
@@ -1,92 +1,126 @@
 package com.wtmberlin.ui
 
 import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.doReturn
+import com.nhaarman.mockitokotlin2.mock
 import com.wtmberlin.data.Repository
 import com.wtmberlin.data.Result
 import com.wtmberlin.data.WtmEvent
-import io.mockk.MockKAnnotations
-import io.mockk.coEvery
-import io.mockk.every
-import io.mockk.impl.annotations.MockK
-import io.mockk.mockk
-import junit.framework.Assert.assertEquals
+import com.wtmberlin.defaultVenue
+import com.wtmberlin.mock
+import com.wtmberlin.util.ErrorLogger
+import com.wtmberlin.util.background
+import com.wtmberlin.util.io
+import com.wtmberlin.util.ui
 import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.test.TestCoroutineDispatcher
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.newSingleThreadContext
 import kotlinx.coroutines.test.setMain
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.Mockito.times
+import org.mockito.Mockito.verify
+import org.mockito.MockitoAnnotations
 import org.threeten.bp.LocalDateTime
 import org.threeten.bp.ZoneId
 import org.threeten.bp.ZonedDateTime
 
+@ExperimentalCoroutinesApi
 class EventsViewModelTest {
     @get:Rule
     val instantTaskExecutorRule = InstantTaskExecutorRule()
 
-    @MockK
-    internal lateinit var repository: Repository
-    private val viewModel by lazy { EventsViewModel(repository) }
-    private val testDispatcher = TestCoroutineDispatcher()
-
-    private val simpleEvent = mockk<WtmEvent>().also {
-        every { it.id } returns "1"
-        every { it.dateTimeStart } returns ZonedDateTime.now()
-        every { it.name } returns "nameEvent"
-        every { it.venue } returns null
-        every { it.meetupUrl } returns "www.google.com"
-    }
+    private val mainThreadSurrogate = newSingleThreadContext("UI thread")
 
-    private fun startEvents(event: WtmEvent = simpleEvent): WtmEvent {
-        coEvery { repository.events() } returns Result(false, listOf(event), null)
-        return event
-    }
+    @Mock
+    private lateinit var mockRepo: Repository
+    @Mock
+    private lateinit var mockErrorLog: ErrorLogger
+    private val viewModel by lazy { EventsViewModel(mockRepo, mockErrorLog) }
 
-    private val simpleUpcomingEvent = mockk<WtmEvent>().also {
-        val dateStart = LocalDateTime.now().plusDays(1)
-        every { it.id } returns "1"
-        every { it.dateTimeStart } returns ZonedDateTime.of(dateStart, ZoneId.systemDefault())
-        every { it.name } returns "nameEvent"
-        every { it.venue } returns null
-        every { it.meetupUrl } returns "www.google.com"
-    }
+    private fun startEvents(event: WtmEvent = mock()): WtmEvent {
+        Mockito.`when`(event.id).thenReturn("1")
```

Comment:
You can also use the `given` … `when` … `then` syntax from `BDDMockito` to avoid protected keywords in Kotlin

Reply:
Done. Thanks for the tip